app/models/manageiq/providers/ansible_tower/inventory/parser/automation_manager.rb

```diff
@@ -31,6 +32,14 @@ def configuration_scripts
       o[:survey_spec] = i.survey_spec_hash
       o[:variables] = i.extra_vars_hash
       o[:inventory_root_group] = target.inventory_groups.lazy_find(i.inventory_id.to_s)
+
+      o[:authentications] = []
+      %w(credential_id cloud_credential_id network_credential_id).each do |credential_attr|
+        next unless i.respond_to?(credential_attr)
```

Comment:
yes, it is.
I can rename it `job_template` - I dont like one letter vars anyway, so dont know what drove me there